pennylane/pulse/transmon.py

```diff
@@ -257,23 +257,30 @@ def transmon_drive(amplitude, phase, freq, wires, d=2):
 
     .. math::
 
-        \Omega(t) \left(e^{i (\phi(t) + \nu t)} b_q + e^{-i (\phi(t) + \nu t)} b^\dagger_q\right)
+        \Omega(t) \sin\left(\phi(t) + \nu t\right) \sum_q Y_q
 
-    where :math:`[b^\dagger_p, b_q] = i \delta_{pq}` are creation and annihilation operators
-    and :math:`q` is the qubit label (``wires``).
+    where :math:`Y_q` are Pauli-Y operators on ``wires`` :math:`\{q\}`.
```

Comment:
```suggestion
    where :math:`Y_q` is the Pauli-Y operator on wire :math:`q`.
```

Reply:
Ah, right, but $Y_q$ is a single operator, no?
Maybe this?
```suggestion
    where :math:`\{Y_q\}` are the Pauli-Y operators on ``wires`` :math:`\{q\}`.
```